index.ts

```diff
@@ -220,14 +226,14 @@ app.prepare().then(() => {
     }
   )
 
-  router.get('/result', async ctx => {
+  router.get('/result', async (ctx: Context) => {
     const packageString = ctx.query.p
-    ctx.redirect(`/package/${packageString.trim()}`)
+    ctx.redirect(`/package/${(packageString as string).trim()}`)
```

Comment:
What is the type of `packageString` if we don't force typecast here?

Reply:
`string | string[] | undefined`

Reply:
We should handle non-string types then (use `invariant` if needed for this)